audonnx/core/model.py

```diff
@@ -226,15 +228,34 @@ def __init__(
     )
     def __call__(
         self,
-        signal: np.ndarray,
-        sampling_rate: int,
+        inputs: np.ndarray | dict[str, object],
```

Comment:
I totally agree that the new name makes sense, but we should have in mind that it might be able to break some existing code, e.g. `model(signal=signal, sampling_rate=sampling_rate)`.
It is very unlikely that anybody used this, but it would be also annoying when `audonnx` leads to errors due to breaking changes.

I'm not decided yet what we should do.